gemelli/preprocessing.py

```diff
@@ -424,34 +336,34 @@ def _construct(self):
             If a conditional-subject pair
             has multiple samples associated
             with it. In this case the
-            default method is to sum them.
+            default method is to mean them.
```

Comment:
Do you plan to support multiple methods of collision resolution? If not, why don't you say `In this case the samples are averaged` to avoid confusion over whether there are multiple options to handle this case.

Reply:
Was planning on it but did not do it in this PR. Moved this to a new issue #17 and will address in a future PR.